apps/web/src/pages/Pools/PoolAdvertisementPage/components/LanguageRequirementDialog.tsx

```diff
@@ -0,0 +1,469 @@
+              <div
+                data-h2-display="base(grid)"
+                data-h2-gap="base(x.5)"
+                data-h2-margin-bottom="base(x1.5)"
+              >
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "To assess your second language proficiency, you will need to complete an evaluation for each of the three skills. After each evaluation, you will obtain a letter code indicating the proficiency level you demonstrated for that skill.",
+                    id: "YW+ZVh",
+                    description:
+                      "Description for second language proficiency on language requirement dialog",
+                  })}
+                </p>
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "Language requirements for bilingual positions are summarized using one of the letter codes for each of the skills. The first letter refers to the required level for <strong>Reading</strong>, the second <strong>Writing</strong> and the third <strong>Speaking</strong>.",
+                    id: "dYejJH",
+                    description:
+                      "Description for second language proficiency on language requirement dialog",
+                  })}
+                </p>
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "In practice, most bilingual positions are either B B B (intermediate) or C B C (advanced).",
+                    id: "+fSWpK",
+                    description:
+                      "Description for second language proficiency on language requirement dialog",
+                  })}
+                </p>
+              </div>
+              <div data-h2-display="base(grid)" data-h2-gap="base(x.5)">
+                <p>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "The Public Service Commission (PSC) is responsible for all second language evaluations for Government of Canada employment opportunities. You can read more about this topic on the PSC's website",
+                    id: "+MCd8X",
+                    description:
+                      "PSC second language evaluations list for language requirements dialog",
+                  }) + intl.formatMessage(commonMessages.dividingColon)}
+                </p>
+                <ul data-h2-display="base(grid)" data-h2-gap="base(x.5)">
+                  <li>
+                    <Link
+                      newTab
+                      external
+                      href={
+                        locale === "fr"
+                          ? "https://www.canada.ca/fr/commission-fonction-publique/services/evaluation-langue-seconde.html"
+                          : "https://www.canada.ca/fr/commission-fonction-publique/services/evaluation-langue-seconde/tests-autoevaluation.html"
```

Comment:
I may be wrong, but  I think the second link is incorrect here. They both go to French pages but each with different content.

Reply:
Oops :sweat_smile: thanks! Fixed by [1622e66](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10628/commits/1622e66e1e1dd91f196e81aafe0b2854a32434f0)